app/helpers/plates_helper.rb

```diff
@@ -34,18 +29,17 @@ def well_properties(well)
   end
 
   def padded_well_name_with_index(plate)
-    ('A'...('A'.getbyte(0) + (plate.size / 12)).chr).each_with_index do |row, ri|
-      (1..12).each_with_index do |col, ci|
+    ('A'...('A'.getbyte(0) + (plate.size / 12)).chr).each_with_index do |row, row_index|
```

Comment:
Not rubocop, but spotted these 'concisely' names variables. Was definitely playing code golf when I wrote some of this.

Reply:
I prefer your change too, clearer